expansions/pandemonium/src/main/java/ladysnake/pandemonium/PandemoniumRequiemPlugin.java

```diff
@@ -75,10 +82,20 @@ public void onRequiemInitialize() {
             }
             return PossessionStartCallback.Result.PASS;
         });
+        CanCurePossessedCallback.EVENT.register((body) -> {
+            if (body.hasStatusEffect(PandemoniumStatusEffects.PENANCE)) {
+                return TriState.FALSE;
+            }
+            return TriState.DEFAULT;
+        });
 
         // Enderman specific behaviour is unneeded now that players can possess them
         PossessionStartCallback.EVENT.unregister(new Identifier(Requiem.MOD_ID, "enderman"));
         PossessionStartCallback.EVENT.register(Pandemonium.id("allow_everything"), (target, possessor, simulate) -> PossessionStartCallback.Result.ALLOW);
+        PossessionStartCallback.EVENT.register(Pandemonium.id("deny_penance_two"), PenanceStatusEffect::canPossess);
+        //noinspection ConstantConditions
```

Comment:
Could you add a comment to note why this suppression is necessary ?

Reply:
Done